survey/src/main/java/com.quickbirdstudios.surveykit/result/question_results/DateTimeQuestionResult.kt

```diff
@@ -0,0 +1,16 @@
+package com.quickbirdstudios.surveykit.result.question_results
+
+import com.quickbirdstudios.surveykit.AnswerFormat
+import com.quickbirdstudios.surveykit.Identifier
+import com.quickbirdstudios.surveykit.result.QuestionResult
+import kotlinx.android.parcel.Parcelize
+import java.util.*
+
+@Parcelize
+data class DateTimeQuestionResult(
+    override val stringIdentifier: String,
+    override val id: Identifier,
+    override val startDate: Date,
+    override var endDate: Date = Date(),
+    val answer: AnswerFormat.DateTimeAnswerFormat.DateTime?
+) : QuestionResult
```

Comment:
Please add a new line at the end ( https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file )